tests/unit/BaseIndexTest.php

```diff
@@ -308,6 +308,21 @@ public function testDoSearch()
         $this->assertInstanceOf(ArrayList::class, $result->getSpellcheck());
         $this->assertGreaterThan(0, $result->getSpellcheck()->count());
         $this->assertNotEmpty($result->getCollatedSpellcheck());
+
+        $index = new CircleCITestIndex();
+        $query = new BaseQuery();
+        $query->addTerm('');
+        $index->doSearch($query);
+
+        $this->assertEquals(['*:*'], $index->getQueryTerms());
+
+        $index = new CircleCITestIndex();
+        $query = new BaseQuery();
+        $query->addTerm('test');
+        $query->addSort('SiteTree_Title', 'asc');
+        $index->doSearch($query);
+
```

Comment:
```suggestion
$this->assertEquals(['test'], $index->getQueryTerms());
```

Reply:
It's a bit a double-up from queries tested above, but indeed, it never hurts to be thorough.